### ISSUE TITLE
Approvals and token_id standards format change

### ISSUE THREAD

**Issue by zcstarr** [Mintbase/mintbase-core#28]
**Summary**
The standard in the NEPs and the standard implemented by the near sdk drifted. This means that a large part of the ecosystem follows the rust sdk standard, satori,pluminite,paras... new project I think. The long and short is id and approvals have been deprecated in favor of 
```
id changed to token_id
approvals changed to approved_account_ids
```
https://github.com/near/NEPs/pull/315

**Motivation**: 
Fix the output of the various methods that return approvals I think primarily `nft_token`. This will make mintbase compatible with the rest of the ecosystem, and the tokens line up for 3rd party integrations, videogames etc... .

**Acceptance criteria:**
 that at minimum token_id and approved_account_ids are returned as fields, mostly things come over the wire as JSON so making an additive change is probably not super problematic, as I think most of this is in the view layer.

**Comment by tifrel** [comment]
Fixed by #31 and already landed on testnet. Gonna close on upcoming mainnet update

**Comment by tifrel** [comment]
Landed on mainnet
